REST: PUT /resources/{resourceGroupName}/{releaseName}/properties/{propertyName} doesn't encrypt property

Setting an encrypted property via  `PUT /resources/{resourceGroupName}/{releaseName}/properties/{propertyName}` doesn't encrypt property. In the database I can see that the property was stored unencrypted.
After setting it, it can't be displayed read or set again as it causes an exception: 
```
2017-07-26 14:54:29,984 logLev=ERROR class=org.apache.catalina.core.ContainerBase.[jboss.web].[default-host].[/AMW_rest].[ch.mobi.itc.mobiliar.rest.RESTApplication] thread="http-/0.0.0.0:8443-20" JBWEB000236: Servlet.service() for servle
t ch.mobi.itc.mobiliar.rest.RESTApplication threw exception: org.jboss.resteasy.spi.UnhandledException: ch.puzzle.itc.mobiliar.common.exception.AMWRuntimeException: Was not able to decrypt properties
        at org.jboss.resteasy.core.SynchronousDispatcher.handleApplicationException(SynchronousDispatcher.java:367)
        at org.jboss.resteasy.core.SynchronousDispatcher.handleException(SynchronousDispatcher.java:235)
        at org.jboss.resteasy.core.SynchronousDispatcher.handleInvokerException(SynchronousDispatcher.java:211)
        at org.jboss.resteasy.core.SynchronousDispatcher.getResponse(SynchronousDispatcher.java:576)
        at org.jboss.resteasy.core.SynchronousDispatcher.invoke(SynchronousDispatcher.java:543)
        at org.jboss.resteasy.core.SynchronousDispatcher.invoke(SynchronousDispatcher.java:128)
        at org.jboss.resteasy.plugins.server.servlet.ServletContainerDispatcher.service(ServletContainerDispatcher.java:208)
        at org.jboss.resteasy.plugins.server.servlet.HttpServletDispatcher.service(HttpServletDispatcher.java:55)
        at org.jboss.resteasy.plugins.server.servlet.HttpServletDispatcher.service(HttpServletDispatcher.java:50)
        at javax.servlet.http.HttpServlet.service(HttpServlet.java:847)
        at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:295)
        at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:214)
        at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:231)
        at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:149)
        at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:512)
        at org.jboss.as.jpa.interceptor.WebNonTxEmCloserValve.invoke(WebNonTxEmCloserValve.java:50)
        at org.jboss.as.jpa.interceptor.WebNonTxEmCloserValve.invoke(WebNonTxEmCloserValve.java:50)
        at org.jboss.as.web.security.SecurityContextAssociationValve.invoke(SecurityContextAssociationValve.java:169)
        at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:150)
        at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:97)
        at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:102)
        at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:344)
        at org.apache.coyote.http11.Http11Processor.process(Http11Processor.java:854)
        at org.apache.coyote.http11.Http11Protocol$Http11ConnectionHandler.process(Http11Protocol.java:653)
        at org.apache.tomcat.util.net.JIoEndpoint$Worker.run(JIoEndpoint.java:926)
        at java.lang.Thread.run(Thread.java:745) [rt.jar:1.8.0_66]
Caused by: ch.puzzle.itc.mobiliar.common.exception.AMWRuntimeException: Was not able to decrypt properties

```

--- a/AMW_business/src/main/java/ch/puzzle/itc/mobiliar/business/property/control/PropertyValueService.java
+++ b/AMW_business/src/main/java/ch/puzzle/itc/mobiliar/business/property/control/PropertyValueService.java
@@ -71,7 +71,7 @@ public class PropertyValueService {
         if (!propertyValidationService.canPropertyValueBeSetOnContext(p.getDescriptor(), resourceContext)) {
             throw new ValidationException("The property " + p.getDescriptor() + " can not be set on context " + resourceContext.getContext());
         }
-        p.setValue(unobfuscatedValue);
+        p.setValueAndEncrypt(unobfuscatedValue);
     }
 
 

--- a/AMW_business/src/test/java/ch/puzzle/itc/mobiliar/business/property/control/PropertyValueServiceTest.java
+++ b/AMW_business/src/test/java/ch/puzzle/itc/mobiliar/business/property/control/PropertyValueServiceTest.java
@@ -128,7 +128,7 @@ public class PropertyValueServiceTest {
         service.setPropertyValue(resourceContextMock, propertyDescriptorId, unobfuscatedValue);
 
         // then
-        Mockito.verify(propertyMock).setValue(unobfuscatedValue);
+        Mockito.verify(propertyMock).setValueAndEncrypt(unobfuscatedValue);
     }
 
     @Test(expected = ValidationException.class)

